### PR TITLE
accept http rpc url

### PR DESCRIPTION
**Pull request by nicolasochem**
It is possible to run rewards distributor on the same machine that runs the tezos node.

Or, like in my case, in a k8s service in the same cluster. In this case, http is fine.

### CODE REVIEW

File: src/config/validate/runtime.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ const validDistributionShares = Joi.custom((i) => {
   }
 });
 
-const validRpcUrl = Joi.string().uri({ scheme: ["https"] });
+const validRpcUrl = Joi.string().uri({ scheme: ["https", "http"] });
```
Comment (cryi) [inline src/config/validate/runtime.ts:44]: Technically any protocol is ok if we run in local setup. So we can add tcp here as well.

**Review by cryi — CHANGES_REQUESTED** [review]
We technically this is not issue at all to support all protocols. We just have to make sure we forge txs with local forger and whether it goes encrypted or not to RPC does not matter really.

**Comment by cryi** [comment]
@kalouo what do you think is best course of action to handle this considering my above comment.

**Comment by kalouo** [comment]
To be honest I am not sure about the different protocols we want to support. 

We can scope it down to what @nicolasochem needs currently and leave it at that for now.

**Review by kalouo — APPROVED** [review]
LGTM

**Pull request merged**